src/Forms/ConfirmedPasswordField.php

```diff
@@ -694,4 +739,18 @@ public function getRequireStrongPassword()
     {
         return $this->requireStrongPassword;
     }
+
+    private function updateRightTitle(): void
```

Comment:
```suggestion
    /**
     * Appends a warning to the right title, or removes that appended warning.
     */
    private function updateRightTitle(): void
```
Just so we don't have to read through the code to know what this method is doing here in the future

Reply:
Updated